code/modules/power/cable.dm

```diff
@@ -573,6 +573,7 @@
 
 	if (length(directions) == 0)
 		cable_laying(0,NORTH)
+		CRASH("Lone cablespawner!\nCable location: [src.x] x, [src.y] y, [src.z] z.")
```

Comment:
oh yeah forgot i did this lol. I think it should be fine, single pipe spawners have a similar crash behaviour, so i thought it should match

Reply:
just going to remove that actually